snap/snapcraft.yaml

```diff
@@ -382,6 +386,11 @@ parts:
             -e s:'http\://localhost\:48061/api/v1/logs':: > \
        "$SNAPCRAFT_PART_INSTALL/config/support-logging/res/configuration.toml"
 
+      cat "./cmd/support-notifications/res/configuration.toml" | \
+        sed -e s:./logs/edgex-support-notifications.log:/var/snap/edgexfoundry/common/support-notifications.log: \
```

Comment:
I guess I never noticed we're doing this replacement here to set the log value to be in `$SNAP_COMMON` at build time. 
I think this is fine as we do it with other services for this PR, but I'd like to file an issue to change this replacement to happen during the `install` hook because on some platforms, such as Arch Linux, $SNAP_DATA and $SNAP_COMMON aren't in the same `/var/snap/edgexfoundry` directories.

Reply:
Good point re: those cross-distro paths.  +1 on moving all of this logic into the install hook.  Note, we do this for mongo-config too.

Reply:
Issue filed for Delhi as https://github.com/edgexfoundry/edgex-go/issues/486